### ISSUE TITLE
RC 2026-02-03

### ISSUE THREAD

**Issue by jordanjones243** [AlaskaAirlines/auro-formkit#1321]
### In this release
- 45230a8 fix: address failing a11y tests in select and combobox
- c6326a2 fix: make menuoptions visible to screenreaders #456
- ad9c2d3 fix: add Chromatic workflow for visual testing on rc branches
- 9afbfbe feat: add story generation utilities and update stories for checkbox component
- 6781a8d fix: adjust types to enable cem to process correctly
  - update playground stories to follow SB v10 format
